cmd/influxd/main.go

```diff
@@ -78,11 +74,13 @@ func main() {
 	case "":
 		execRun(args)
 	case "backup":
+		setGoMaxProcs()
```

Comment:
@otoolep looks like we can remove it from everywhere but run. However, the reason I didn't put it in the switch statement for the run is it printed above the logo, and felt out of place.  I'll just leave that in the run and remove all of them from the switch, as well as the actual function now that we are clear on the use case.